mongodbatlas/clusters.go

```diff
@@ -395,6 +408,31 @@ func (s *ClustersServiceOp) GetProcessArgs(ctx context.Context, groupID, cluster
 	return root, resp, err
 }
 
+// Status gets the status of the operation on the Cluster.
+// See more: https://docs.atlas.mongodb.com/reference/api/clusters-check-operation-status/
+func (s *ClustersServiceOp) Status(ctx context.Context, groupID, clusterName string) (ClusterStatus, *Response, error) {
+	if err := checkClusterNameParam(clusterName); err != nil {
+		return ClusterStatus{}, nil, err
+	}
+
+	basePath := fmt.Sprintf(clustersPath, groupID)
+	escapedEntry := url.PathEscape(clusterName)
+	path := fmt.Sprintf("%s/%s/status", basePath, escapedEntry)
+
+	req, err := s.Client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return ClusterStatus{}, nil, err
+	}
+
+	root := new(ClusterStatus)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return ClusterStatus{}, resp, err
+	}
+
+	return *root, resp, err
```

Comment:
[nit] I guess you don't want to deal with pointers (not going to judge on that) but you could reuse the empty object a bit more like
```suggestion
	var root ClusterStatus
	req, err := s.Client.NewRequest(ctx, http.MethodGet, path, nil)
	if err != nil {
		return root, nil, err
	}

	resp, err := s.Client.Do(ctx, req, &root)
	return root, resp, err
```

would that work for your case (not blocking on this if it doesn't work for what you're going here)?

Reply:
yes, this looks better, thanks! 👍